src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -742,4 +748,285 @@ public void receiveSimpleMessageFromPartitionAndOffset(TestContext context) {
 
         deleteAsync.await();
     }
+
+    @Ignore
+    @Test
+    public void sendToOnePartitionTest(TestContext context) {
+        String kafkaTopic = "sendToOnePartitionTest";
+        kafkaCluster.createTopic(kafkaTopic, 3, 1);
+
+        Async async = context.async();
+
+        String value = "Hi, This is kafka bridge";
+        int partition = 1;
+
+        JsonArray records = new JsonArray();
+        JsonObject json = new JsonObject();
+        json.put("value", value);
+        records.add(json);
+
+        JsonObject root = new JsonObject();
+        root.put("records", records);
+
+        WebClient client = WebClient.create(vertx);
+
+        client.post(BRIDGE_PORT, BRIDGE_HOST, "/topics/" + kafkaTopic + "/partitions/" + partition)
+                .putHeader("Content-length", String.valueOf(root.toBuffer().length()))
+                .as(BodyCodec.jsonObject())
+                .sendJsonObject(root, ar -> {
+                    context.assertTrue(ar.succeeded());
+
+                    HttpResponse<JsonObject> response = ar.result();
+                    JsonObject bridgeResponse = response.body();
+
+                    JsonArray offsets = bridgeResponse.getJsonArray("offsets");
+                    context.assertEquals(1, offsets.size());
+                    JsonObject metadata = offsets.getJsonObject(0);
+                    context.assertNotNull(metadata.getInteger("partition"));
+                    context.assertEquals(partition, metadata.getInteger("partition"));
+                    context.assertEquals(0L, metadata.getLong("offset"));
+                    async.complete();
+                });
+    }
+
+    @Ignore
+    @Test
+    public void partitionPriorityTest(TestContext context) {
+        // partition is specified in the message body and in the POST request
+        // what does have higher priority? Lets say POST one
```

Comment:
you are right that the priority is on the URL because the client is doing a POST to a specific endpoint but I would return an error about wrong payload because you cannot specify the "partition" in the JSON. Pushing silently, can confuse the user which thinks that we pushed the message to the partition in the JSON while it went to the one in the URL.